projects/plugins/backup/src/js/components/Admin.js

```diff
@@ -89,7 +89,7 @@ const Admin = () => {
 	const renderLoadedState = () => {
 		// Loading state
 		if ( ! connectionLoaded ) {
-			return renderJetpackConnection();
+			return renderConnectScreen();
```

Comment:
We can avoid displaying the screen while loading the connection by wrapping this in a hidden div.
```
return (
  <div style={ { 'display': 'none' } }>
    { renderConnectScreen() }
  </div>
);
```
We can probably also just remove the call to `renderPromptForConnection` entirely and replace it with `renderConnectScreen` in the `! connectionStatus.isUserConnected || ! connectionStatus.isRegistered` state.

Reply:
That was an oversight on my part.
I temporarily fixed that in #20355, and will implement the more elegant solution a bit later.